src/modules/tools/zprobe/CartGridStrategy.cpp

```diff
@@ -175,7 +188,8 @@ bool CartGridStrategy::handleConfig()
         return false;
     }
 
-    reset_bed_level();
```

Comment:
This makes absolutely no sense at all. It is RAM, it is totally irrelevant whether you read or write to RAM. Why do you believe this is necessary? What is the advantage?

I recommend you submit these as two separate requests as the manual probe stuff I may consider but this reducing memory writes is nonsense IMO.

Reply:
It is not RAM.  We allocate memory in the program memory block.  It is Flash if I am not mistaken?
If you do not clear it, the values remain even after resetting.

Reply:
It is RAM, AHB0 is RAM! Exactly the Same as main RAM, it is just one of the three non contiguous blocks of RAM in he LPC1769.

Reply:
Interesting.
Please ignore this pull request then.  I will do more faultfinding to find the cause of my problems

Reply:
I will separate the two features

Reply:
FWIW the only time we ever write to flash is when you flash a new firmware binary which I believe you could do 100,000 times before wearing out the memory. (it may be 10,000 but still more than anyone is likely to do). Also you cannot randomly write to flash, you must first erase a block then write a block (I think a block is 4k bytes).